docs/source/qvm.rst

```diff
@@ -186,7 +186,7 @@ QPU lattice for a particular time block.
 When your reservation begins, you will be authorized to access the QPU. A configuration file will be
 automatically populated for you with the proper QPU endpoint for your reservation. Both your QMI and the QPU
```

Comment:
QMI's no longer exist - can refer to JupyterLab notebook instead (which is not on-premise but has similar characteristics).

Reply:
Good catch. I did a search within all the docs sources for "QMI" and corrected a number of other places too.